speedtest/test_performance_openssl.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+"""
+A py.test test that attempts to build openssl and benchmark the effect of clcache
+"""
+import sys
+import os
+import shutil
+import pytest
+import urllib
+import zipfile
+import subprocess
+import time
+
+
+OPENSSL_ZIP = "OpenSSL_1_0_2-stable.zip"
+OPENSSL_URL = "https://codeload.github.com/openssl/openssl/zip/OpenSSL_1_0_2-stable"
+
+THISDIR = os.path.dirname(os.path.abspath(__file__))
+DISTDIR = os.path.join(os.path.dirname(THISDIR), "dist")
+CLCACHE = os.path.join(DISTDIR, "cl.exe")
+SOURCES = []
```

Comment:
Why is `SOURCES` a list? It appears the code always only accesses the first element of it?